docker_for_ctp/ctp_config/ha_repl.conf

```diff
@@ -0,0 +1,83 @@
+# Copyright (c) 2016, Search Solution Corporation. All rights reserved.
```

Comment:
please modify the license headers for all `*.conf` files

Reply:
Thank you.
I will reflect on it.